tests/test_schemas.py

```diff
@@ -106,35 +107,35 @@ def getRandomValue(self, cls, fieldName):
         """
         maxListSize = 10
         field = self.getAvroSchema(cls, fieldName)
-        t = field.type
-        if isinstance(t, avro.schema.UnionSchema):
+        typ = field.type
+        if isinstance(typ, avro.schema.UnionSchema):
             # TODO put in some probability that return None if
             # we have a union schema
-            t = t.schemas[1]
-        if isinstance(t, avro.schema.MapSchema):
+            typ = typ.schemas[1]
+        if isinstance(typ, avro.schema.MapSchema):
             ret = {}
             for j in range(random.randint(0, maxListSize)):
-                l = [randomString() for j in range(
+                randStr = [randomString() for j in range(
```

Comment:
Yes, it should be.

Something like pylint would catch these things, but I don't know if we want to add that to our build system... pep8 is picky enough!  It's something to consider, however.